EIPS/eip-1.md

```diff
@@ -265,11 +265,15 @@ The `description` field in the preamble:
 
 When referring to an EIP by number, it should be written in the hyphenated form `EIP-X` where `X` is the EIP's assigned number.
 
+### IETF RFCs
+
+When referring to an IETF RFC by number, it must be written in the non-hyphenated form `RFC X`, where `X` is the RFC's assigned number. Each RFC that is referenced in an EIP **MUST** be accompanied by a relative markdown link to the RFC (in the `assets/eip-1/RFCs` directory) the first time it is referenced, and **MAY** be accompanied by a link on subsequent references. If the RFC is not currently in that directory, a separate pull request must be submitted which only includes the RFC in that directory.
```

Comment:
Why a separate pull request?

Reply:
I would prefer if the git history wasn't polluted with random unrelated PRs. @MicahZoltu's the editor that's probably going to have the strongest opinion on this front.